src/components/modules/MainPage/LongCard/StyledLongCard.js

```diff
@@ -0,0 +1,66 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+  width:100rem;
+  height: 17.5rem;
+  height:100%;
+  margin: 0.5%;
+  font-size:1.2rem;
+  border-bottom-right-radius:40px;
+  border-bottom-left-radius:40px;
+  background-color:#C3C0C0;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23); */
+  display: flex;
+  flex-direction: column;
+  @media (orientation: portrait) {
+    width: 100%;
+    height:100%;
```

Comment:
height of 100% doesn't do anything usually, it can be deleted

Reply:
deleted